overlord/snapstate/storehelpers.go

```diff
@@ -69,6 +70,28 @@ func userFromUserID(st *state.State, userIDs ...int) (*auth.UserState, error) {
 	return user, err
 }
 
+func refreshOptions(st *state.State, origOpts *store.RefreshOptions) (*store.RefreshOptions, error) {
+	var opts store.RefreshOptions
+
+	if origOpts != nil {
+		if origOpts.RequestSalt != "" {
+			// nothing to add
+			return origOpts, nil
+		}
+		opts = *origOpts
+	}
+
+	if err := st.Get("refresh-request-salt", &opts.RequestSalt); err != nil && err != state.ErrNoState {
```

Comment:
I wonder if we should rename it to something more generic, we could reuse it for all sorts of salting probably.

Reply:
what I mean  a persistent stable salt is not what one would usually use as a salt, salts are normally operation to make them unique, non clashing

Reply:
Indeed, given that this is a very unusual salt I wonder if we should simply call it something different. Maybe just `refresh-request-secret` or `-key` or something?

Reply:
secret gives the wrong idea, we have too many things called key already.   refresh-privacy-key perhaps ?  (leaving out request because is getting too long)